### PR TITLE
Add case export on node

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

**Review by FranckLecuyer — APPROVED** [review]

File: src/translations/en.json [hunk 1]
```diff
@@ -253,6 +253,7 @@
     "createNetworkModificationNode": "Create a network modification node",
     "createModelNode": "Create a model node",
     "removeNode": "Remove node",
+    "exportNode": "Export node",
```
Comment (SlimaneAmar) [inline src/translations/en.json:256]: It is not the node that is exported but the associated network

Comment (sBouzols) [reply src/translations/en.json]: New proposal

Comment (SlimaneAmar) [reply src/translations/en.json]: Export network ?

Comment (sBouzols) [reply src/translations/en.json]: Export Case PO approval

**Pull request merged**